<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionar Proyectos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../imgs/logo1.png</normaloff>../imgs/logo1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0" colspan="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_insertar">
           <property name="font">
            <font>
             <family>Bahnschrift Light SemiCondensed</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
background-color:rgb(225, 81, 81);
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;}

QPushButton:hover{
    background-color: rgb(113, 40, 40);}</string>
           </property>
           <property name="text">
            <string>Insertar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_actualizar">
           <property name="styleSheet">
            <string notr="true">QPushButton{
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
background-color:rgb(225, 81, 81);
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;}

QPushButton:hover{
  background-color: rgb(113, 40, 40);}</string>
           </property>
           <property name="text">
            <string>Actualizar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_eliminar">
           <property name="styleSheet">
            <string notr="true">QPushButton{
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
background-color:rgb(225, 81, 81);
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;}

QPushButton:hover{
  background-color: rgb(113, 40, 40);}</string>
           </property>
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cerrar">
           <property name="styleSheet">
            <string notr="true">QPushButton{
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
background-color:rgb(225, 81, 81);
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;}

QPushButton:hover{
  background-color: rgb(113, 40, 40);}</string>
           </property>
           <property name="text">
            <string>Cerrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">margin: 5px;
font: 13pt &quot;Bahnschrift&quot;;</string>
        </property>
        <property name="text">
         <string>Proyectos</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">margin: 10px 10px 5px 4px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Nombre del Proyecto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">margin: 10px 10px 5px 5px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Línea de Investigación</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">margin: 10px 10px 5px 5px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Tiempo de duración(Meses)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="txt_nombre_proyecto">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin:0 5px 5px 5px;
outline:none;
border:1px solid #aaa;
padding:7px;
border-radius: 5px;
color: #333;
}

QLineEdit:focus{
border: 1px solid rgb(225, 81, 81);

}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Ingrese nombre del Proyecto</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="rbtn_no">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{font: 7pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 0 5px 0 5px;


padding:5px;
    border-radius:          10px;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          10px;
padding:5px;
}

QRadioButton::indicator:checked {
    background-color:      rgb(225, 81, 81);

}

QRadioButton::indicator:unchecked {
    background-color:       #f3f3f3;
border-radius: 10px;
    border:                 1px solid #bbbbbb;
}
</string>
        </property>
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="rbtn_si">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{font: 7pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 0 5px 0 5px;



padding:5px;
    border-radius:          10px;
}

QRadioButton::indicator {
    width:                  10px;
    height:                 10px;
    border-radius:          10px;
padding:5px;
}

QRadioButton::indicator:checked {
    background-color:      rgb(225, 81, 81);

}

QRadioButton::indicator:unchecked {
    background-color:       #f3f3f3;
border-radius: 10px;
    border:                 1px solid #bbbbbb;
}</string>
        </property>
        <property name="text">
         <string>Si</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">margin: 10px 10px 5px 5px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Internacional</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txt_linea_investigacion">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin:0 5 5 5;
outline:none;
border:1px solid #aaa;
padding:7px;
border-radius: 5px;
color: #333;

}

QLineEdit:focus{
border: 1px solid rgb(225, 81, 81);

}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Ingrese línea de investigación</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="txt_tiempo_duracion">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin:0 5 5 5;
outline:none;
border:1px solid #aaa;
padding:7px;
border-radius: 5px;
color: #333;
}

QLineEdit:focus{
border: 1px solid rgb(225, 81, 81);

}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Ingrese el tiempo de duración en meses</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QTableWidget" name="tabla_proyectos">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section{
font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
}

QTableWidget QtableCornerButton::section{
font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
border: solid 1px black;


}
QTableWidget{font: 8pt &quot;Bahnschrift Light SemiCondensed&quot;;
border:solid 1px gray;
margin:20px 0px 10px 120px;
}
</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">margin: 10px 10px 5px 5px;
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
        </property>
        <property name="text">
         <string>Jefe del Proyecto</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="txt_jefe_proyecto">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin:0 5 5 5;
outline:none;
border:1px solid #aaa;
padding:7px;
border-radius: 5px;
color: #333;
}

QLineEdit:focus{
border: 1px solid rgb(225, 81, 81);

}
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Ingrese el Jefe de Proyecto</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
